src/Core/Datatypes/ColorMap.cc

```diff
@@ -150,6 +150,42 @@ namespace detail
     virtual Core::Datatypes::ColorRGB getColorMapVal(double v) const override;
   };
 
+  class Turbo : public ColorMapStrategy
```

Comment:
Add comment with URL for the source of all these new colormaps.

Reply:
https://github.com/matplotlib/matplotlib/blob/master/LICENSE/LICENSE

Reply:
and the google one

Reply:
I added the link to the license for matplotlib. Is that what yall wanted?

Reply:
Sure